selenium_tests/pin/page_objects/pin_page.py

```diff
@@ -0,0 +1,66 @@
+
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.common.by import By
+from selenium_tests.pin.page_objects.base_page_object import PinBasePageObject
+import abc
+
```

Comment:
Usually there are two blank lines between classes in a `.py` file, and between imports and the first `class`, etc. (PyCharm will show you this if you're editing in PyCharm, we can look at it together if you wish.)